### PR TITLE
[8.0] porting product_customer_code picking

### PR DESCRIPTION
**Pull request by hurrinico**
Port module product_customer_code to 8.0

### CODE REVIEW

File: product_customer_code_picking/README.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Nicola Malcontenti <nicola.malcontenti@agilebg.com>
+* Alex Comba <alex.comba@agilebg.com>
```
Comment (eLBati) [inline product_customer_code_picking/README.rst:35]: Why did you remove me? :innocent:

Comment (hurrinico) [inline product_customer_code_picking/README.rst:35]: ops :D fixed

File: product_customer_code_picking/__openerp__.py [hunk 2]
```diff
@@ -31,19 +31,9 @@
...
-This module makes the product customer code visible in the stock moves
-of a picking.
-
-Contributors
-------------
-* Nicola Malcontenti <nicola.malcontenti@agilebg.com>
-* Alex Comba <alex.comba@agilebg.com>
-* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
-""",
     "demo": [],
```
Comment (tafaRU) [inline product_customer_code_picking/__openerp__.py:34]: Please remove empty key.

**Comment by Cedric-Pigeon** [comment]
@hurrinico  could you check why runbot is red ?

**Comment by eLBati** [comment]
Following https://github.com/OCA/product-attribute/pull/122#issuecomment-304813468 I would close this
